products/views.py

```diff
@@ -43,6 +43,25 @@ def get(self, request):
 
         return JsonResponse({'products':products_lst}, status=200)
 
+class ProductView(View):
+    def get(self, request, product_id):
+        if not Product.objects.filter(id=product_id).exists():
+            return JsonResponse({"MESSAGE":"INVALID_ID"}, status=404)
+
+        product       = Product.objects.get(id=product_id)
+        product_data = {
```

Comment:
```suggestion
        product      = Product.objects.get(id=product_id)
        product_data = {
```
사소한 것이지만 항상 확인해 주세요.